test/DefaultCluster.Tests/CollectionFixtures.cs

```diff
@@ -1,11 +1,12 @@
-﻿using TestExtensions;
+﻿using DefaultCluster.Tests.General;
+using TestExtensions;
 using Xunit;
 
 namespace DefaultCluster.Tests
 {
     // Assembly collections must be defined once in each assembly
     [CollectionDefinition("DefaultCluster")]
-    public class DefaultClusterTestCollection : ICollectionFixture<DefaultClusterFixture> { }
+    public class DefaultClusterTestCollection : ICollectionFixture<DefaultClusterFixture>, ICollectionFixture<DefaultClusterTestsFixture> { }
```

Comment:
hmmm, not sure I follow, these should create both fixtures, doesn't it? essentially starting 2 different clusters?